src/Avalonia.Input/Platform/PlatformHotkeyConfiguration.cs

```diff
@@ -73,8 +75,12 @@ public PlatformHotkeyConfiguration() : this(KeyModifiers.Control)
             {
                 new KeyGesture(Key.End, commandModifiers | selectionModifiers)
             };
+            OpenContextMenu = new List<KeyGesture>
+            {
+                new KeyGesture(Key.Apps)
```

Comment:
I believe this gesture is expected to be cross platform.
But still on macOS it's not handled. And as I found there is no keyboard gesture to open context menu on macOS.

Reply:
I think at least on windows there's a message for it: [`WM_CONTEXTMENU`](https://docs.microsoft.com/en-us/windows/win32/menurc/wm-contextmenu) so maybe the gesture should come from the platform backend?

Doesn't need to be done in this PR though.

Reply:
Yes, we can support it in the future